From 3a41a4bb8d8f9f5fe7f7bf57e025dc30aa04b32d Mon Sep 17 00:00:00 2001
From: Patrick Rohr <prohr@google.com>
Date: Fri, 24 Mar 2023 10:45:27 -0700
Subject: [PATCH] Rename Cronet API classes from org.chromium.net to
 android.net.http.

Also clean up the API surface a bit.

Test: m
Change-Id: I12c0ca0829e005f291eda7c690dfa401d91987b9
---
 components/cronet/android/BUILD.gn | 68 +++++++++++++++---------------
 1 file changed, 34 insertions(+), 34 deletions(-)

diff --git a/components/cronet/android/BUILD.gn b/components/cronet/android/BUILD.gn
index 699ca663ae488..464167b7f120b 100644
--- a/components/cronet/android/BUILD.gn
+++ b/components/cronet/android/BUILD.gn
@@ -236,39 +236,39 @@ sizes_test("cronet_sizes") {
 # cronet_api_java.jar defines Cronet API.
 android_library("cronet_api_java") {
   sources = [
-    "api/src/org/chromium/net/BidirectionalStream.java",
-    "api/src/org/chromium/net/CallbackException.java",
-    "api/src/org/chromium/net/ConnectionMigrationOptions.java",
-    "api/src/org/chromium/net/CronetEngine.java",
-    "api/src/org/chromium/net/CronetException.java",
-    "api/src/org/chromium/net/DnsOptions.java",
-    "api/src/org/chromium/net/ExperimentalBidirectionalStream.java",
-    "api/src/org/chromium/net/ExperimentalCronetEngine.java",
-    "api/src/org/chromium/net/ExperimentalUrlRequest.java",
-    "api/src/org/chromium/net/ICronetEngineBuilder.java",
-    "api/src/org/chromium/net/InlineExecutionProhibitedException.java",
-    "api/src/org/chromium/net/NetworkException.java",
-    "api/src/org/chromium/net/NetworkQualityRttListener.java",
-    "api/src/org/chromium/net/NetworkQualityThroughputListener.java",
-    "api/src/org/chromium/net/QuicException.java",
-    "api/src/org/chromium/net/QuicOptions.java",
-    "api/src/org/chromium/net/RequestFinishedInfo.java",
-    "api/src/org/chromium/net/UploadDataProvider.java",
-    "api/src/org/chromium/net/UploadDataSink.java",
-    "api/src/org/chromium/net/UrlRequest.java",
-    "api/src/org/chromium/net/UrlResponseInfo.java",
-    "api/src/org/chromium/net/apihelpers/ByteArrayCronetCallback.java",
-    "api/src/org/chromium/net/apihelpers/ContentTypeParametersParser.java",
-    "api/src/org/chromium/net/apihelpers/CronetRequestCompletionListener.java",
-    "api/src/org/chromium/net/apihelpers/CronetResponse.java",
-    "api/src/org/chromium/net/apihelpers/ImplicitFlowControlCallback.java",
-    "api/src/org/chromium/net/apihelpers/InMemoryTransformCronetCallback.java",
-    "api/src/org/chromium/net/apihelpers/JsonCronetCallback.java",
-    "api/src/org/chromium/net/apihelpers/RedirectHandler.java",
-    "api/src/org/chromium/net/apihelpers/RedirectHandlers.java",
-    "api/src/org/chromium/net/apihelpers/StringCronetCallback.java",
-    "api/src/org/chromium/net/apihelpers/UploadDataProviders.java",
-    "api/src/org/chromium/net/apihelpers/UrlRequestCallbacks.java",
+    "api/src/android/net/http/BidirectionalStream.java",
+    "api/src/android/net/http/CallbackException.java",
+    "api/src/android/net/http/ConnectionMigrationOptions.java",
+    "api/src/android/net/http/CronetEngine.java",
+    "api/src/android/net/http/CronetException.java",
+    "api/src/android/net/http/DnsOptions.java",
+    "api/src/android/net/http/ExperimentalBidirectionalStream.java",
+    "api/src/android/net/http/ExperimentalCronetEngine.java",
+    "api/src/android/net/http/ExperimentalUrlRequest.java",
+    "api/src/android/net/http/ICronetEngineBuilder.java",
+    "api/src/android/net/http/InlineExecutionProhibitedException.java",
+    "api/src/android/net/http/NetworkException.java",
+    "api/src/android/net/http/NetworkQualityRttListener.java",
+    "api/src/android/net/http/NetworkQualityThroughputListener.java",
+    "api/src/android/net/http/QuicException.java",
+    "api/src/android/net/http/QuicOptions.java",
+    "api/src/android/net/http/RequestFinishedInfo.java",
+    "api/src/android/net/http/UploadDataProvider.java",
+    "api/src/android/net/http/UploadDataSink.java",
+    "api/src/android/net/http/UrlRequest.java",
+    "api/src/android/net/http/UrlResponseInfo.java",
+    "api/src/android/net/http/apihelpers/ByteArrayCronetCallback.java",
+    "api/src/android/net/http/apihelpers/ContentTypeParametersParser.java",
+    "api/src/android/net/http/apihelpers/CronetRequestCompletionListener.java",
+    "api/src/android/net/http/apihelpers/CronetResponse.java",
+    "api/src/android/net/http/apihelpers/ImplicitFlowControlCallback.java",
+    "api/src/android/net/http/apihelpers/InMemoryTransformCronetCallback.java",
+    "api/src/android/net/http/apihelpers/JsonCronetCallback.java",
+    "api/src/android/net/http/apihelpers/RedirectHandler.java",
+    "api/src/android/net/http/apihelpers/RedirectHandlers.java",
+    "api/src/android/net/http/apihelpers/StringCronetCallback.java",
+    "api/src/android/net/http/apihelpers/UploadDataProviders.java",
+    "api/src/android/net/http/apihelpers/UrlRequestCallbacks.java",
     _generated_interface_api_version_java,
   ]
 
@@ -1622,4 +1622,4 @@ if (!is_component_build) {
       ]
     }
   }
-}
\ No newline at end of file
+}
-- 
2.40.0.348.gf938b09366-goog

